Makefile

```diff
@@ -134,24 +134,23 @@ license-check:
 
 FFMPEG_ROOT ?= etc/FFmpeg
 $(FFMPEG_ROOT):
-	cd etc && git clone https://github.com/FFmpeg/FFmpeg.git
-	git -C $(FFMPEG_ROOT) checkout release/6.1
+	cd etc && git clone https://github.com/FFmpeg/FFmpeg.git --depth 1 --branch release/6.1
 
 # For ARM64 builds, use the image ghcr.io/viamrobotics/antique:arm64 for backward compatibility
-FFMPEG_H264_PREFIX ?= $(shell realpath .)/gostream/codec/h264/ffmpeg/$(shell uname -s)-$(shell uname -m)
-ffmpeg-h264-static: $(FFMPEG_ROOT)
-	cd $(FFMPEG_ROOT) && ./configure \
-		--disable-programs \
-		--disable-doc \
-		--disable-everything \
-		--enable-encoder=h264_v4l2m2m \
-		--prefix=$(FFMPEG_H264_PREFIX) \
-		--enable-pic
+FFMPEG_PREFIX ?= $(shell realpath .)/gostream/ffmpeg/$(shell uname -s)-$(shell uname -m)
+FFMPEG_OPTS = --disable-programs --disable-doc --disable-everything --prefix=$(FFMPEG_PREFIX) --enable-pic --disable-autodetect
```

Comment:
[info] I had not looked into the `PIC` linking flag before. Found GNU reference [here](https://gcc.gnu.org/onlinedocs/gcc/Code-Gen-Options.html#index-fpic).

Reply:
Great catch! I removed this since I don't think we need it. Everything works fine without it.